packages/wrangler/src/deploy/deploy.ts

```diff
@@ -786,6 +791,12 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 						}
 					);
 
+					// Update tail consumers and logpush settings
+					await patchNonVersionedScriptSettings(accountId, scriptName, {
```

Comment:
Can/should we run the deploy and patch requests in parallel?

Reply:
Since this does a deploy we'd have concurrency issues sadly where they'd try to block on eachother and probably more often than not timeout